content/en/examples/controllers/pod-rs.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
```

Comment:
I think this file needs to go in content/en/examples/pods. Otherwise the [example verification test](https://travis-ci.org/kubernetes/website/jobs/486021005) fails with this: examples_test.go:576: controllers/pod-rs.yaml: pod-rs does not have a test case defined